src/platforms/dotnet/guides/aspnet/index.mdx

```diff
@@ -39,6 +39,7 @@ public class MvcApplication : System.Web.HttpApplication
         // Add this to Application_Start
         _sentry = SentrySdk.Init(o =>
         {
+            o.AddAspNet();
             o.Dsn = new Dsn("___PUBLIC_DSN___");
```

Comment:
This also changed.
`Dsn` class is internal and a String is passed only. Needs fixing.